components/Officers.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Row, Col } from "react-bootstrap";
-import { OfficersArray } from "./data/Officers";
-import { Colors } from "./data/Colors";
+import OfficersArray from "./data/Officers";
```

Comment:
rename array to Officers

Reply:
yes, but leaving in Colors (but using default export) cuz thats used in the file 
![image](https://user-images.githubusercontent.com/12877445/193467405-4dd22580-74c4-4011-bb2e-2d6612329f9d.png)

Reply:
oh whops didnt see the Colors below (comment only showed above snippet)